NEWS.md

```diff
@@ -1,44 +1,30 @@
 # rsconnect (development version)
 
-* `deployDoc()` and `deployApp()` now support deploying static content to Posit
-  Cloud. Static RMarkdown and Quarto content can be rendered server-side.
-
-* When recording details about deployments to Posit Cloud, appId now represents
-  the content id (as seen in URLs of the format 
-  `https://posit.cloud/content/{id}`) instead of the application id.
-
-* A `version` field has been added to deployment DCF files to facilitate file
-  format changes. Its value for this release is `1`.
-
-* `deployApp()` now gives an actionable error if you attempt to set
-  visiblity of an app deployed to posit.cloud (#838).
+## New features
 
-* `deploySite()` now supports quarto websites (#813).
+* `deployApp()` and `deployDoc()` now support deploying static content to Posit
+  Cloud. Static RMarkdown and Quarto content can be rendered server-side.
 
 * `deployApp()` and `writeManifest()` now respect renv lock files, if present. 
   If you don't want to use these lockfiles, and instead return the previous 
   behaviour of snapshotting on every deploy, add your `renv.lock` to 
   `.rscignore` (#671). Learn more `?appDependencies()`.
   
-* `deployApp()` and `writeManifest()` now use renv to capture app dependencies, 
-  rather than packrat. If this causes a previously working deploy to fail, 
-  please file an issue then set `options(rsconnect.packrat = TRUE)` to revert 
-  to the previous behaviour.
-
-* The built-in linter should have fewer false positives for path problems:
-  the relative path linter has been removed (#244) and the case-sensitive 
-  linter now only checks strings containing a `/` (#611).
+  Additionally, `deployApp()` and `writeManifest()` now use renv to capture app 
+  dependencies,  rather than packrat. If this causes a previously working deploy 
+  to fail, please file an issue then set `options(rsconnect.packrat = TRUE)` to 
+  revert to the previous behaviour.
 
 * `deployApp()`'s `quarto` argument now takes values `TRUE`, `FALSE` or 
   `NA`. The previous value (a path to a quarto binary) is now deprecated,
-  and instead we automatically figure out the packge from `QUARTO_PATH` and
-  `PATH` env vars (#658).
+  and instead we automatically figure out the package from `QUARTO_PATH` and
```

Comment:
Calling the `quarto=<string>` usage "deprecated" feels too soft. The value is ignored and implies `quarto=TRUE`.